.reuse/dep5

```diff
@@ -0,0 +1,24 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Nextcloud Vue components library
+Upstream-Contact: Nextcloud GmbH <https://nextcloud.com/impressum/>
+Source: https://github.com/nextcloud-libraries/nextcloud-vue
+
+Files: package.json package-lock.json .github/pull_request_template.md
+Copyright: Nextcloud GmbH and Nextcloud contributors
+License: AGPL-3.0-or-later
+
+Files: styleguide/assets/* cypress/snapshots/* tests/unit/functions/usernameToColor/__snapshots__/usernameToColor.spec.js.snap
+Copyright: Nextcloud GmbH and Nextcloud contributors
```

Comment:
I'm not sure all the files in this folder have this license. app-background, for example, or twitter logo.

Reply:
Adjusted most of them, removed the unused other